src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidatorBuilder.cs

```diff
@@ -82,6 +125,30 @@ public IMarkdownTokenRewriter Create()
                 });
         }
 
+        private IEnumerable<MarkdownTagValidationRule> GetEnabledTagRules()
+        {
+            foreach (var item in _tagValidators)
+            {
+                if (item.FullId != null)
+                {
+                    MarkdownValidationRule rule;
+                    if (_validators.TryGetValue(item.FullId, out rule) ||
```

Comment:
In new design, move this to `settings`

---
In reply to: [72934361](https://github.com/dotnet/docfx/pull/545#discussion_r72934361) [](ancestors = 72934361)